src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -372,7 +373,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
                 for (const auto& m : marks)
                 {
                     const auto row = m.Start.Y;
-                    const til::color color{ m.Color.Color };
+                    const COLORREF color{ til::color{ m.Color.Color } };

```

Comment:
We are still converting it to a COLORREF just to convert it back in the call to drawPip. Is that intentional?